connectors/connectors/rest/rest-swagger-connector/src/main/resources/camel-connector.json

```diff
@@ -0,0 +1,31 @@
+{
+  "baseScheme": "rest-swagger",
+  "baseGroupId": "org.apache.camel",
+  "baseArtifactId": "camel-rest-swagger",
+  "baseVersion": "2.20.0",
+  "baseJavaType" : "org.apache.camel.component.rest.swagger.RestSwaggerComponent",
+  "name": "rest-swagger-connector",
+  "scheme": "swagger-operation",
+  "javaType": "io.syndesis.connector.swagger.SwaggerConnectorComponent",
+  "groupId": "io.syndesis",
+  "artifactId": "rest-swagger-connector",
+  "version": "0.5-SNAPSHOT",
```

Comment:
Is this version also generated ? Really confusing that generated classes / files are committed.

If so, could you please update to 1.2-SNAPSHOT ?